modules/nf-core/kallisto/quant/main.nf

```diff
@@ -16,10 +16,10 @@ process KALLISTO_QUANT {
     val fragment_length_sd
 
     output:
-    tuple val(meta), path("${prefix}") , emit: results
-    tuple val(meta), path("*info.json"), emit: json_info
-    tuple val(meta), path("*.log.txt") , emit: log
-    path "versions.yml"                , emit: versions
+    tuple val(meta), path("${prefix}")                    , emit: results
+    tuple val(meta), path("${prefix}/run_info.json")      , emit: json_info
+    tuple val(meta), path("${prefix}/kallisto_quant.log") , emit: log
```

Comment:
There are known issues with providing a directory as output as well as it's content. Don't think we can use this approach and will have to revert to making a copy of the file.